como-conocer-necesidades-usuarios.md

```diff
@@ -0,0 +1,56 @@
+CÓMO CONOCER LAS NECESIDADES DE TUS USUARIOS
+
+INTRODUCCIÓN
+
+Actualmente la interacción se ha vuelto fundamental para que la relación usuario-empresa no muera. Debido a ello, es necesario centrarse en la observación del desarrollo de las necesidades actuales del ser humano. Esto es importante para lograr que el usuario tenga una experiencia satisfactoria con el producto. Para lograr esa satisfacción del usuario, hay que definir a quien va dirigido el producto, es decir, cual será el público objetivo. 
+
+Una vez definido el público, debemos identificar cuál es la técnica más apropiada para que puedas desarrollarla según los objetivos del producto.  A continuación, se van a explicar algunas de las técnicas que se pueden utilizar:
+
+TIPOS DE TÉCNICAS
+
+Skype como herramienta(ANGIE)
```

Comment:
La persona asignada a cada sección es una información útil mientras trabajáis, pero no debería incluirse para la versión final.